### PR TITLE
Migrate all apps to run on k8s

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  name: bancho-service
+  environment: production # TODO: move this out of values.yaml
+  group: backend
+  replicaCount: 1
+  container:
+    image:
+      repository: registry.digitalocean.com/akatsuki/bancho-service
+      tag: latest
+    imagePullSecrets:
+      - name: akatsuki
```
Comment (cmyui) [inline chart/values.yaml]:
```suggestion
      - name: registry-akatsuki
```

File: Dockerfile [hunk 2]
```diff
@@ -4,6 +4,7 @@ ENV PYTHONUNBUFFERED=1
 
 COPY requirements.txt .
 RUN pip install -r requirements.txt
+RUN pip install -i https://pypi2.akatsuki.gg/cmyui/dev akatsuki-cli
```
Comment (cmyui) [inline Dockerfile]:
```suggestion
RUN pip install git+https://github.com/osuAkatsuki/akatsuki-cli
```

File: .github/workflows/k8s-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
+name: k8s-deploy
+
+on:
+  push:
+    branches:
+      - main
+      - k8s-deploy
```
Comment (cmyui) [inline .github/workflows/k8s-deploy.yml]:
```suggestion
```

File: chart/values.yaml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      env:
+        - name: APP_COMPONENT
+          value: api
+      imagePullSecrets:
+        - name: osuakatsuki-registry-secret
+    service:
+      type: ClusterIP
+      port: 80
+
+  - name: bancho-service-spam-protection-cronjob
```
Comment (cmyui) [inline chart/values.yaml:36]: These crons don't have good handling of SIGTERM; todo

**Review by tsunyoku — DISMISSED** [review]

**Review by tsunyoku — APPROVED** [review]

**Pull request merged**